WalletWasabi.Fluent/Helpers/StartupHelper.cs

```diff
@@ -10,6 +11,12 @@ public static class StartupHelper
 	{
 		private const string KeyPath = "SOFTWARE\\Microsoft\\Windows\\CurrentVersion\\Run";
 
+		// Arguments to add Wasabi to macOS startup settings.
+		private static readonly string AddArguments = $"-c \"osascript -e \' tell application \\\"System Events\\\" to make new login item at end of login items with properties {{name:\\\"{nameof(WalletWasabi)}\\\", path:\\\"/Applications/WasabiWallet.app\\\",hidden:false}} end tell\' \"";
```

Comment:
This is basically impossible to read, I would add

```csharp
<remarks><code>unescaped command here</code></remarks>
```

or ortherwise tried to simplify it because at this point it's very hard to read or modify.

Reply:
Comment added. I can't further refactor the arguments, every word is needed.